components/AppFooter/AppFooter.tsx

```diff
@@ -0,0 +1,164 @@
+import { createStyles, Text, Container, ActionIcon, Group } from '@mantine/core';
+import AppLink from '../AppLink/AppLink';
+import Image from "next/image";
+import LogoSmall from "../../public/logo_small.png";
+import { DiscordIcon } from "../AppHeader/DiscordIcon";
+import { SteamIcon } from "./SteamIcon";
+import { FooterTextLinks } from "./FooterTextLinks"
+
+const useStyles = createStyles((theme) => ({
+  footer: {
+    marginTop: 120,
+    paddingTop: theme.spacing.xl * 2,
+    paddingBottom: theme.spacing.xl * 2,
+    backgroundColor: theme.colorScheme === 'dark' ? theme.colors.dark[6] : theme.colors.gray[0],
+    borderTop: `1px solid ${
+      theme.colorScheme === 'dark' ? theme.colors.dark[5] : theme.colors.gray[2]
+    }`,
+  },
+
+  logo: {
+    maxWidth: 200,
+
+    [theme.fn.smallerThan('sm')]: {
+      display: 'flex',
+      flexDirection: 'column',
+      alignItems: 'center',
+    },
+  },
+
+  description: {
+    marginTop: 5,
+
+    [theme.fn.smallerThan('sm')]: {
+      marginTop: theme.spacing.xs,
+      textAlign: 'center',
+    },
+  },
+
+  inner: {
+    display: 'flex',
+    justifyContent: 'space-between',
+
+    [theme.fn.smallerThan('sm')]: {
+      flexDirection: 'column',
+      alignItems: 'center',
+    },
+  },
+
+  groups: {
+    display: 'flex',
+    flexWrap: 'wrap',
+
+    [theme.fn.smallerThan('sm')]: {
+      display: 'none',
+    },
+  },
+
+  wrapper: {
+    width: 160,
+  },
+
+  link: {
+    display: 'block',
+    color: theme.colorScheme === 'dark' ? theme.colors.dark[1] : theme.colors.gray[6],
+    fontSize: theme.fontSizes.sm,
+    paddingTop: 3,
+    paddingBottom: 3,
+
+    '&:hover': {
+      textDecoration: 'underline',
+    },
+  },
+
+  title: {
+    fontSize: theme.fontSizes.lg,
+    fontWeight: 700,
+    fontFamily: `Greycliff CF, ${theme.fontFamily}`,
+    marginBottom: theme.spacing.xs / 2,
+    color: theme.colorScheme === 'dark' ? theme.white : theme.black,
+  },
+
+  afterFooter: {
+    display: 'flex',
+    justifyContent: 'space-between',
+    alignItems: 'center',
+    marginTop: theme.spacing.xl,
+    paddingTop: theme.spacing.xl,
+    paddingBottom: theme.spacing.xl,
+    borderTop: `1px solid ${
+      theme.colorScheme === 'dark' ? theme.colors.dark[4] : theme.colors.gray[2]
+    }`,
+
+    [theme.fn.smallerThan('sm')]: {
+      flexDirection: 'column',
+    },
+  },
+
+  social: {
+    [theme.fn.smallerThan('sm')]: {
+      marginTop: theme.spacing.xs,
+    },
+  },
+}));
+
+const AppFooter = () => {
+  const { classes } = useStyles();
+  const groups = FooterTextLinks.map((group) => {
+    const links = group.links.map((link, index) => (
+      <Text<'a'>
+        key={index}
```

Comment:
It's not the best practice to use `index` as `key`:
https://reactjs.org/docs/lists-and-keys.html#keys
Even if this shouldn't be a real issue here (because the array order won't change), I would prefer to use `link.link` as `key`, which should be unique too.